test_values/cache_tests.json

```diff
@@ -137,6 +145,191 @@
             "02242b37d8e851f1e86f46790298c7097df06893d6226b7c1453c213e91717de",
             "df12e87db0413ef51d5bbd374904eeeb5cc4615ddae80e45a7140983f30cd01c"
         ],
-        "to_destroy": [24]
+        "to_destroy": [24],
+        "new_del_pos": [],
+        "new_del_hashes": []
+    },
+    {
+        "initial_stump": "Add six leaves [0, 1, 2, 3, 4, 5]",
+        "roots": [
+            "df46b17be5f66f0750a4b3efa26d4679db170a72d41eb56c3e4ff75a58c65386",
+            "9eec588c41d87b16b0ee226cb38da3864f9537632321d8be855a73d5616dcc73"
+        ],
+        "leaves": 6,
+        "additional_preimages": [0, 1, 2, 3, 4],
+        "del_hashes": [
+            "6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d",
+            "4bf5122f344554c53bde2ebb8cd2b7e3d1600ad631c385a5d7cce23c7785459a",
+            "dbc1b4c900ffe48d575b5da5c638040125f65db0fe3e24494b76ea986457d986",
+            "084fed08b978af4d7d196a7446a86b58009e636b611db16211b65a9aadff29c5"
+        ],
+        "proof_hashes": [
```

Comment:
The verification for this proof will pass since it needs no proof hashes to verify `[0, 1, 2, 3]` in a forest with 6 leaves. I'm not sure if we should treat these as a failure but since putting in any hashes here will pass the tests.

But we should really do it the correct way and have nothing here as the proofs.

Reply:
I'm not sure where this hashes came from. I removed them.